frontend/src/components/Footer.vue

```diff
@@ -0,0 +1,17 @@
+<template lang="pug">
+  v-footer(color="#000080" padless)
```

Comment:
Define any new color in https://github.com/devlup-labs/portfolio/blob/master/frontend/src/plugins/vuetify.js